cuda_pathfinder/docs/source/api.rst

```diff
@@ -6,7 +6,8 @@
 ``cuda.pathfinder`` API Reference
 =================================
 
-The ``cuda.pathfinder`` module provides utilities for loading NVIDIA dynamic libraries.
+The ``cuda.pathfinder`` module provides utilities for loading NVIDIA dynamic libraries,
+and experimental APIs for locating NVIDIA C/C++ header directories.
 
 Public API
 -----------
```

Comment:
nit: while we're here, I'd like to remove this heading because it is visually unpleasant; we already have a main heading above
```suggestion
```

Reply:
Done: commit 410d866d136605291544a68935398955d3cd8b57

I tried it out locally: it definitely looks better after that commit.